chains/abstract/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.abscan.org/api
+    family: etherscan
+    name: Abstract Block Explorer
+    url: https://abscan.org
+blocks:
+  confirmations: 1
+  estimateBlockTime: 1
+  reorgPeriod: 0
```

Comment:
remind me where we landed w/ reorg periods for zksync chains?

Reply:
still 0 i believe, all the other zksync chains are still set to this too